spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -290,6 +327,13 @@ def then_i_should_see_correct_targeted_delivery_funding_breakdown
   end
 
   def then_i_should_see_the_correct_total
+    within first(".app-application__card") do
+      expect(page).to have_content("Course total")
+      expect(page).to have_content(number_to_pounds(1_458))
```

Comment:
although we are testing the final total, this doesn't fail with the previous implementation. Could we add a test that fails if we were to include vat?

Reply:
With the vat related code removed, i'm not sure if we need to do this test